src/libertem/udf/stddev.py

```diff
@@ -1,56 +1,133 @@
-import collections
-
 import numpy as np
+import numba
 
 from libertem.udf import UDF
 
 
-VariancePart = collections.namedtuple('VariancePart', ['var', 'sum_im', 'N'])
+@numba.njit(fastmath=True)
```

Comment:
Is the `fastmath` needed here? Is that to give the compiler additional freedom when inlining this function in the other `merge`?

Reply:
Some tests revealed that `fastmath=True` only on merge_single brings the biggest speed-up. I have no idea why.

Reply:
I optimized the function further with blocking. With that version, fastmath for both `merge_single` and `process_tile` works best.